Automattic/jetpack
# Add SIG feature flag check to use-publicize-config

This adds the check to the Initial State on the UI side for the SIG feature flag.

### Other information:


## Jetpack product discussion



1203876111187910-as-1203969267014316

## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






Test this with its' corresponding wpcom change: D101773-code
Sandbox that patch, and check out this change. In `projects/plugins/social/src/js/editor.js` import in the `isSocialImageGeneratorEnabled` check and log check its value with and without the `publicize-social-image-generator` blog sticker.
